src/StudentManagementApplication.java

```diff
@@ -1,10 +1,7 @@
-import model.Student;
 import service.StudentServiceImpl;
-import java.util.ArrayList;
-import java.util.List;
 import java.util.Scanner;
 
-public class Main {
+public class StudentManagementApplication {
```

Comment:
Line 9 is redundant
line 12 should add 1 space after 1. (eg. 1. Show...)
line 13 the same update with line 12
line 19 - not proper variable name. student -> studentService
StudentServiceImpl student = new StudentServiceImpl(); -> StudentService studentService = new StudentServiceImpl();
line 27. The same update with line 19